docs/survey.md

```diff
@@ -1,36 +1,40 @@
 # .NET Library and Open Source Survey
 
-## When you're building class libraries, you're mostly doing this because
+## What is the primary reason why you're building class libraries?
 
-- ( ) You need to break down your own application logic
-- ( ) You need re-usable components that are used by other people's apps
-- ( ) You rarely build libraries
+- ( ) I break down my application logic
+- ( ) I build re-usable components that are used by other people's apps
+- ( ) I build them for fun
+- ( ) I rarely build libraries
 
-## When you're building libraries, you're mostly building for
+## How often do you target multiple frameworks for a single library?
 
-- ( ) a single target framework
-- ( ) multiple target frameworks
+- ( ) Very often
+- ( ) Often
+- ( ) Sometimes
+- ( ) Rarely
+- ( ) Very rarely
 
-## When you're consuming libraries, do you often see issues because different parts of your dependencies only work with specific versions of the library?
+## How often do you experience issues because different parts of your dependencies only work with specific versions of a library?
 
-- ( ) Strongly agree
-- ( ) Somewhat agree
-- ( ) Neither agree nor disagree
-- ( ) Somewhat disagree
-- ( ) Strongly disagree
+- ( ) Very often
+- ( ) Often
+- ( ) Sometimes
+- ( ) Rarely
+- ( ) Very rarely
 
-## Do you own NuGet packages that are available on nuget.org?
+## Do you own NuGet packages on nuget.org?
 
 - ( ) Yes
 - ( ) No
 
-## What describes your open source activity best?
+## What describes your open source activity best (check all that applies)?
+
+**Filter: This will show the "As an XXX question blocks"**
 
-- ( ) I'm mostly maintaining open source projects
-- ( ) I'm maintaining but also frequently contributing to other open source projects
-- ( ) I'm frequently contributing to open source projects
-- ( ) I'm mostly consuming open source projects
-- ( ) I don't use open source projects
+- [ ] I'm maintaing open source projects
+- [ ] I'm contributing to open source projects
```

Comment:
How about this? Because that's also a big difference (although both are useful)
```suggestion
- [ ] I'm contributing code to open source projects
- [ ] I'm contributing docs/samples to open source projects
- [ ] I'm contributing by helping people use open source projects (StackOverflow, GitHub Discussions, ...)
```

Reply:
I deliberately didn't say code because I consider participation in discussions or adding/fixing documentation valueable too.